Sources/Get/Request.swift

```diff
@@ -11,22 +11,22 @@ import FoundationNetworking
 /// An HTTP network request.
 public struct Request<Response>: @unchecked Sendable {
     /// HTTP method, e.g. "GET".
-    public var method: String
+    public var method: HTTPMethod
     /// Resource URL. Can be either absolute or relative.
-    public var url: String
+    public var url: URL?
```

Comment:
Is there a motive for why this is optional? I would expect that the developer would make sure that the request is valid to begin with. In the instance of the `init(path: String ...)` we could either:
1 - throw an error with this init, however I don't like having this init throwable and the other not for consistency
2 - just crash at runtime and make the developer worry about it

Reply:
The initializer that accepts `path` as a `String` can fail to produce a `URL`:

```swift
public init(path: String, method: HTTPMethod = .get, ...) {
    self.url = URL(string: path) // Optional
```